lib/ParamedicPermissions.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+/* jshint node: true */
+var path 	 = require('path');
+var fs 		 = require('fs');
+var shelljs  = require("shelljs");
+var logger  = require('./utils').logger;
+
+function ParamedicPermissions(appName, simulatorsFolder, dbPath) {
+	this.appName = appName;
+	this.simulatorsFolder = simulatorsFolder;
+	this.dbPath = dbPath;
+}
+
+ParamedicPermissions.prototype.updatePermissions = function(serviceList){
+	var self = this;
+	this.getDirectories(self.simulatorsFolder).forEach(function(simFolder){
+		var destinationTCCFile = path.join(self.simulatorsFolder, simFolder, '/data/Library/TCC/TCC.db');
+
+		//Check if the simFolder has TCC.db file in it
+		if(self.doesFileExist(destinationTCCFile)) {
+			var sqlite3Command_firstPart  = 'sqlite3 ' + destinationTCCFile;
+			var sqlite3Command_secondPart = ' "insert into access(service, client, client_type, allowed, prompt_count, csreq) values(\'';
+			var sqlite3Command_lastPart   = '\',\'' + self.appName + '\',0,1,1,NULL)"';
+			serviceList.forEach(function(service){
+				var command = sqlite3Command_firstPart  +  sqlite3Command_secondPart + service + sqlite3Command_lastPart ;
+				logger.info("Running Command: " + command);
+				shelljs.exec(command, {silent: true, async: false});
+			});
+		} else {
+			// No TCC.db file exists by default. So, Copy the new TCC.db file
+			var destinationTCCFolder = path.join(self.simulatorsFolder, simFolder, '/data/Library/TCC');
+			if(!self.doesFileExist(destinationTCCFolder)){
+				fs.mkdir(destinationTCCFolder, 0777);
```

Comment:
Please use 0644 instead of 0777 here. Also, please factor out the permissions setting into a variable.